Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -99,9 +98,14 @@ public final class SwiftPackageManagerGraphGenerator: SwiftPackageManagerGraphGe
             }
 
             let packageInfo = try swiftPackageManagerController.loadPackageInfo(at: packageFolder)
+            let artifactPaths = workspaceState.object.artifacts.filter { artifact in
+                artifact.packageRef.identity == dependency.packageRef.identity
+            }.map { AbsolutePath($0.path) }
```

Comment:
nit: I fint this a bit more readable
```suggestion
            let artifactPaths = workspaceState.object.artifacts
                .filter { $0.packageRef.identity == dependency.packageRef.identity }
                .map { AbsolutePath($0.path) }
```